cmd/translatesfx/translatesfx/cli.go

```diff
@@ -0,0 +1,55 @@
+package translatesfx
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"gopkg.in/yaml.v2"
+)
+
+// CLI is the entry point for the translatecfg command.
+func CLI(args []string) {
+	translateConfig(paths(args))
+}
+
+// paths returns two arguments from the command line args:
+// cfgFname is the path to the Smart Agent config that is being translated
+// wd is the working directory used to evaluate file paths found in the Smart Agent config
+func paths(args []string) (cfgFname string, wd string) {
+	switch len(args) {
+	case 2:
+		wd, err := os.Getwd()
+		if err != nil {
+			log.Fatalf("error getting working directory: %v", err)
+		}
+		return args[1], wd
+	case 3:
+		return args[1], args[2]
+	default:
+		fmt.Println("usage: translatesacfg <path/to/smart/agent/config.yaml>")
```

Comment:
`log.Fatal()` so that we don't push this down to a failed file read?

Reply:
Oh yeah, that's a good point. Will fix.

Reply:
Done